### ISSUE TITLE
Safenet token - missing `modules: string[]` property in `SmartCard`

### ISSUE THREAD

**Issue by SuperITMan** [Trust1Team/t1c-sdk-js#8]
**Is your feature request related to a problem? Please describe.**

In order to display the right certificates/tokens, I'd like to use the property `modules` available in my Safenet token.

```json
{
    "id": "1******a",
    "name": "AKS ifdh 0",
    "pinpad": false,
    "card": {
        "atr": "29*****************d8",
        "description": [
            "Safenet Token"
        ],
        "modules": [
            "safenet"
        ]
    }
}
```

**Describe the solution you'd like**

Adapt the interface `SmartCard` like this:

```typescript
export declare class SmartCard {
    atr?: string | undefined;
    description?: string[] | undefined;
    modules?: string[] | undefined;
    constructor(atr?: string | undefined, description?: string[] | undefined, modules?: string[] | undefined);
}
```

Regards,

Alexis GEORGES

**Comment by GillesPl** [comment]
Hi Alexis, 
Thanks for the ticket.

will be fixed in 3.6.1

**Comment by SuperITMan** [comment]
Hi @GillesPl,

Could you confirm that the sdk-js v3.6.1 will be compatible with the API v3.5.20 ?

Kr,
Alexis
